apps/rush-lib/src/cli/utilities/TaskManager.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  RushConfiguration,
+  RushConfigurationProject,
+  IRushLinkJson
+} from '../../index';
+import { JsonFile } from '@microsoft/node-core-library';
+
+import TaskRunner from '../taskRunner/TaskRunner';
+import ProjectBuildTask from '../taskRunner/ProjectBuildTask';
+
+export class TaskManager {
```

Comment:
Document this:

Why we need it?
Who calls it and when?
How is it implemented?